src/conversions/std/time.rs

```diff
@@ -0,0 +1,189 @@
+//! Conversions here do not rely on the floating point timestamp of the timestamp/fromtimestamp APIs
```

Comment:
I wonder, is it possible there exists a fast-path for cases where the timestamp doesn't fall into a range where the loss of precision is a potential problem? I think Python datetimes only carry microsecond accuracy, I'd expect there is a range where `f64` can express this without error.

Reply:
Replied in the main discussion.